### PR TITLE
Add protofile modal design integration

### PR DESCRIPTION
**Pull request by sonicyeti**
This PR covers the design integration of the gRPC add proto file modal. 

**Notes:**
- Selected state differs from default hover
- Delete icon border removed > set to "danger" color
- All other changes per design spec

**Example:**

**Additional Story for "bordered" instance:**
Screen Shot 2020-11-09 at 10 22 39 AM

Fixes INS-246

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/base/file-input-button.js [hunk 1]
```diff
@@ -91,7 +92,11 @@ class FileInputButton extends React.PureComponent<Props> {
         title={path}
         {...(extraProps: Object)}>
         {showFileIcon && <i className="fa fa-file-o space-right" />}
-        {showFileName && fileName ? `${fileName}` : `Choose ${name || 'File'}`}
+        {showFileName && fileName
+          ? `${fileName}`
+          : staticLabel && staticLabel.length
+          ? staticLabel
+          : `Choose ${name || 'File'}`}
```
Comment (develohpanda) [inline packages/insomnia-app/app/ui/components/base/file-input-button.js]: I feel like the `staticLabel` can take precedence here, so something like:

```js
{staticLabel || (showFileName && fileName ? `${fileName}` : `Choose ${name || 'File'}`)}
```

What do you think? Not really fussed either way.

File: packages/insomnia-app/app/ui/components/modals/proto-files-modal.js [hunk 2]
```diff
@@ -118,13 +118,13 @@ class ProtoFilesModal extends React.PureComponent<Props, State> {
       <Modal ref={this._setModalRef}>
         <ModalHeader>Select Protofile</ModalHeader>
         <ModalBody className="wide pad">
-          <div className="row-spaced">
+          <div className="row-spaced margin-bottom bold">
             Files
             <FileInputButton
-              name=".proto file"
-              showFileIcon
+              showFileName="false"
```
Comment (develohpanda) [inline packages/insomnia-app/app/ui/components/modals/proto-files-modal.js]: With my previous comment, `showFileName` can be omitted here

File: packages/insomnia-components/components/list-group/list-group.js [hunk 3]
```diff
@@ -4,16 +4,24 @@ import styled from 'styled-components';
 
 type Props = {|
   children?: React.Node,
+  bordered?: boolean,
```
Comment (develohpanda) [inline packages/insomnia-components/components/list-group/list-group.js:7]: 🔥

**Review by develohpanda — APPROVED** [review]
Looking gooooooooood! 🎉

File: packages/insomnia-app/app/ui/components/base/file-input-button.js (hunk 1 above)
Comment (sonicyeti) [reply packages/insomnia-app/app/ui/components/base/file-input-button.js]: This is easier to read as well, going to test where we are adding certs, config etc. The gRPC implementation is the only case where we have a fixed static label. If working, we'll rock this for sure.

Comment (sonicyeti) [reply packages/insomnia-app/app/ui/components/base/file-input-button.js]: Working well and back up, thanks!

**Review by develohpanda — APPROVED** [review]

**Comment by erickrico** [comment]
looks great, good job 🔥

**Pull request merged**